packages/connection-form/src/hooks/use-connect-form.ts

```diff
@@ -289,13 +312,55 @@ function handleUpdateHost({
   }
 }
 
+function isConnectionUpdatePersonalisationAction(
+  action: ConnectionFormFieldActions
+): action is UpdateConnectionPersonalisationAction {
+  return action.type === 'update-connection-personalisation';
+}
+
+export function handleConnectionFormUpdateForPersonalisation(
+  action: ConnectionFormFieldActions,
+  personalisation: ConnectionPersonalisationOptions,
+  connectionString?: string
+): ConnectionPersonalisationOptions {
+  if (!isConnectionUpdatePersonalisationAction(action)) {
+    if (!personalisation.isNameDirty && connectionString) {
+      try {
+        const parsedConnString = new ConnectionString(connectionString);
+        const name = parsedConnString.hosts.join(',');
+        return {
+          ...personalisation,
+          name,
+        };
+      } catch (ex) {
+        // just keep previous value
```

Comment:
`debug()` or `log.debug()` or something?

Reply:
Maybe, but it's not a `real error`. This might happen every time the user is typing a character until the URL is correct, so I'm not sure is very relevant to log this. But if you think it is, I'll add a logger there.